docs/release-notes/1.3.0/release-1.3.0.md

```diff
@@ -0,0 +1,97 @@
+# [ML.NET](http://dot.net/ml) 1.3.0
+
+## **New Features**
+- **Deep Neural Networks Training (PREVIEW)** ([#4057](https://github.com/dotnet/machinelearning/pull/4057))  
+  Introduces `Microsoft.ML.DNN` package that enables full DNN model retraining and transfer learning in .NET using C# bindings for tensorflow provided by Tensorflow .NET. The goal of this package is to allow high level DNN training and scoring tasks such as image classification, text classification, object detection, etc using simple yet powerful APIs that are framework agnostic but currently they only uses Tensorflow at the backend. The below APIs are in early preview and we hope to get customer feedback that we can incorporate in the next iteration.
+
+  ![DNN stack](dnn_stack.jpg)
+
+  ```cs
```

Comment:
Do we need to show this amount of code in the RELEASE NOTES? In the past releases we usually never had this kind of code or signature code in the RELEASE NOTES. It feels cleaner and more concise if we don't put code as part of the release notes, I think. But links pointing to code examples for each feature is usually what we had. 👍

Reply:
Pytorch and TF do it quite a bit and even show diagrams ...

Reply:
Right, this point can be arguable, of course. But TF and PyTorch target a very different profile (it is low level DNN APIs mostly used by data scientists, not .NET developers) than ML.NET which is mostly about .NET developers.

See .NET Core release notes samples:
https://github.com/dotnet/machinelearning-samples/projects/1 

See previous ML.NET release notes samples where users would expect to continue to be consistent with being concise in the release notes, too:

https://github.com/dotnet/machinelearning/blob/master/docs/release-notes/1.0.0-preview/release-1.0.0-preview.md

I think diagrams are good, though. But code/signatures, it usually will expand the release notes too much. That kind of details should be explained in further provided links, I think.